ui/webui/README.rst

```diff
@@ -8,9 +8,44 @@ Make sure you have `npm` available (usually from your distribution package).
 These commands check out the source and build it into the `dist/` directory::
 
     git clone https://github.com/rhinstaller/anaconda.git
+    cd anaconda
+    ./autogen.sh && ./configure
     cd ui/webui
     make
 
+Note that this builds only the WebUI code. In most cases, the rest of Anaconda is needed too.
+
+Running the UI interactively
+----------------------------
+
+First, build RPM packages in the CI container, from root of the repository::
+
+    make -f ./Makefile.am anaconda-ci-build
```

Comment:
Instead of adding all these container and make commands here could we just link to https://github.com/rhinstaller/anaconda/blob/master/CONTRIBUTING.rst#how-to-run-make-commands ?